src/main/java/seedu/address/logic/LogicManager.java

```
@@ -87,9 +88,22 @@ public synchronized CommandResult execute(String commandText)
     @Override
     public synchronized void eagerEvaluate(String commandText, Consumer<String> displayResult) {
         final Command command = addressBookParser.eagerEvaluateCommand(commandText);
+        String currCommandWord = commandText.substring(0, commandText.indexOf(' ') + 1);
+        if (!lastCommandWord.equals(currCommandWord)) {
+            displayResult.accept("");
+            try {
+                this.execute(lastCommandWord);
+            } catch (CommandException | ParseException e) {
+                // do nothing
```

Please avoid having empty catch blocks. Atleast log the exception.